### ISSUE TITLE
Can I get the confidence value for the image?

### ISSUE THREAD

**Issue by 23hyun** [mk-minchul/AdaFace#147]
How can I get confidence values for images?

**Comment by afm215** [comment]
Hello, are you talking about getting the output logits for the classification task?
